Web_Development/FrontEnd/CSS_web_fonts.md

```diff
@@ -0,0 +1,73 @@
+# CSS WEB FONTS
+
+CSS Web Fonts allow us to use fonts that are not installed locally. After choosing the the desired font, include the font file on the web server and you are good to go.
+This font can be defined within the CSS ` @font-face ` rule.
```

Comment:
Not the exact correct definition. Web fonts are more than this, please include more information & also mention about importing Google Fonts into HTML & CSS file